Fundamentals/Setup/Install/index.md

```diff
@@ -32,6 +32,8 @@ Learn how to run an already installed local installation of Umbraco.
 
 .NET CLI, included with the .NET SDK, can be used to install or uninstall .NET templates from NuGet using the `dotnet new` command on any OS. The underlying Template Engine enables the creation of custom templates which make new project bootstrapping much faster. With a few steps you can have an Umbraco project running without the need for a code editor.
 
+There is a useful website which makes installation of Umbraco and the associated packages a lot easier for you. Go to [https://psw.codeshare.co.uk](https://psw.codeshare.co.uk), configure your options and click on the `Install Script` tab to get the commands you need to paste into the terminal. This even includes the commands for unattended install, which creates the database for you and can install a starter kit too, if you wish.
```

Comment:
```suggestion
:::tip
The Community Team has developed some packages which you can find at [https://psw.codeshare.co.uk](https://psw.codeshare.co.uk). On the website, configure your options to generate the required script to run. Click on the `Install Script` tab to get the commands you need to paste into the terminal. This tab also includes the commands for a starter kit or unattended install which creates the database for you.
:::
```

Reply:
Hi Esha, doing it in a tip is a great idea. Your changes to the content though has changed the meaning of it.

Can we update the text to be this:

:::tip

Members of the Umbraco Community have created a website which you can find at [https://psw.codeshare.co.uk](https://psw.codeshare.co.uk). The website makes the installation of Umbraco packages a lot easier for you. On the website you can configure your options to generate the required script to run. Click on the `Install Script` tab to get the commands you need to paste into the terminal. This tab also includes the commands for adding a starter kit or unattended install which creates the database for you.

:::